bindings/python/test.py

```diff
@@ -7,6 +7,11 @@
 import os
 import shutil
 import binascii
+import subprocess as sub
+import sys
+
+if os.name != 'nt':
+	import pty
```

Comment:
You can't use the default terminal for multible parallel client.py executions, we need to create a new pty for them and pty does not work on windows.